### PR TITLE
Feature/83 extend summary

### PR DESCRIPTION
**Pull request by kostobog**
@blcham

### CODE REVIEW

File: src/main/java/cz/cvut/kbss/analysis/dao/FaultTreeDao.java [hunk 1]
```diff
@@ -43,8 +45,10 @@ public EntityDescriptor getEntityDescriptor(FaultTree entity) {
     @Override
     public EntityDescriptor getEntityDescriptor(URI uri) {
         EntityDescriptor entityDescriptor = new EntityDescriptor(uri);
+        super.setEntityDescriptor(entityDescriptor);
         EntityType<FaultTree> ft = em.getMetamodel().entity(type);
         EntityType<FaultEvent> fe = em.getMetamodel().entity(FaultEvent.class);
+//        EntityType<FaultEventType> fet = em.getMetamodel().entity(FaultEventType.class);
```
Comment (blcham) [inline src/main/java/cz/cvut/kbss/analysis/dao/FaultTreeDao.java:51]: Why need it ?

**Review by blcham — APPROVED** [review]

**Comment by blcham** [comment]
@kostobog  docker CI failed

File: src/main/java/cz/cvut/kbss/analysis/dao/FaultTreeDao.java (hunk 1 above)
Comment (kostobog) [reply src/main/java/cz/cvut/kbss/analysis/dao/FaultTreeDao.java]: Removed

**Comment by kostobog** [comment]
Failure of docker image build fixed.

**Pull request merged**